package.json

```diff
@@ -18,11 +18,15 @@
   "author": "Stephen Rugh <rugh@adobe.com>",
   "license": "Apache-2.0",
   "dependencies": {
-    "apollo-link": "~1.2.13"
+    "@apollo/client": "~3.3.15",
+    "react": "^17.0.2",
+    "react-refresh": "^0.10.0",
```

Comment:
Seems like apollo client expects you to have react and react-refresh. Without these dependencies (even as peerDeps) I could not get pwa studio to run.